src/components/Dashboard/FilingHistoryList/bodies/FutureEffective.vue

```diff
@@ -72,8 +72,8 @@ export default class FutureEffective extends Vue {
     if (EnumUtilities.isTypeContinuationIn(this.filing)) {
       return {
         subtitle: 'Future Effective Incorporation Date',
-        filingLabel: 'continuation in',
-        filingTitle: 'Continuation In Application'
+        filingLabel: 'continuation application',
```

Comment:
As with my previous comment, check how this is displayed. Maybe we just want to say "continuation" and omit "application" (same as line 54).

While you're here, do you think the filing titles (lines 55, 69 and 76) should use the FilingNames enum values?

Reply:
Same issue as the previous comment, can't access the filing history page now, but looking at old screenshots, it seems the label and subtitle all go to the last default value on line 79:
![image](https://github.com/bcgov/business-filings-ui/assets/122495122/016e044a-9386-4812-b23a-3d810d94f2eb)

![image](https://github.com/bcgov/business-filings-ui/assets/122495122/ca80cebc-c3d3-4004-9274-0d45820b6cb6)
 I agree we can use the FilingNames enum values

Reply:
Updated, according to Andy's design